market/src/main/resources/application.yml

```
@@ -36,4 +36,4 @@ mybatis:
 security:
   jwt:
     secret-key: testtesttesttesttesttesttesttesttesttesttest
-    expire-length: 360000
+    expire-length: 36000000
```

```suggestion
    expire-millis: 36000000
```
이처럼 이름에 단위를 함께 쓰면 설정할 때에 실수를 방지 할 수 있습니다.